src/handwritten/task.py

```diff
@@ -349,9 +365,36 @@ def close(self):
             return
 
         self._interpreter.clear_task(self._handle)
```

Comment:
If you close the gRPC channel or shut down the server, `clear_task` raises an error and this code doesn't clean up the event handlers, so `__del__` reports a warning. I'll fix it and add tests in a follow-up PR.

Reply:
#400